kolibri/plugins/learn/assets/src/views/ContentUnavailablePage.vue

```diff
@@ -39,6 +42,12 @@
         return this.isLearner && !this.canManageContent;
       },
     },
+    beforeMount() {
+      if (plugin_data.isSubsetOfUsersDevice) {
+        const deviceContentUrl = urls['kolibri:kolibri.plugins.device:device_management'];
+        redirectBrowser(deviceContentUrl());
```

Comment:
We should add two conditionals here:
```
if (this.canManageContent && deviceContentUrl) {
```

No point redirecting them if the user does not have content management permissions, and if the plugin is disabled, then the `deviceContentUrl` will be undefined.

Reply:
Also, is this required? We redirect a user to the device page after initial setup, and we already have a URL for them to be navigate there.

Reply:
Honestly, I'm not satisfied with this solution either, but it was the only way I found to do it for the case where the setup wizard uses an admin account to sync the users.
In that case, after the wizard finishes, it shows the login screen (the wizard can't sign in with the first synced user because it does not know the user password), and after login the user is redirected to learn, not to device. 


As you said, the user is redirected to device after initial setup... if setup syncing is done by the users not but the admin.

Reply:
Confirmed that this PR shows the modal @jredrejo describes for learners that were imported using an admin account.

Seems reasonable to me to keep this with the introduction of the extra conditions, and preferably also adding a comment explaining why this is necessary

Reply:
Yeah, I think I can live with this with the extra conditions and the comment - the explanation for why this is necessary helps!

Reply:
There's another option, @jtamiace I'd like to know your opinion: the main problem comes from the fact that using the admin account to import users we don't know the users' passwords. When importing finishes the wizard logouts the admin user so any user can login and will be redirected to learn.

If the wizard does not logs out the admin, after the wizard finishes kolibri will be open under the admin account in the device section and all the code from `ContentUnavailablePage.vue `can be removed, it's not needed as first view of kolibri will be automatically in the device page . If being logged as an admin user after the wizard finishes it is a good option I can remove several lines of code because the wizard will finish always with the same behaviour, no matter who did the sync.

Reply:
> If the wizard does not logs out the admin, after the wizard finishes kolibri will be open under the admin account in the device section and all the code from ContentUnavailablePage.vue can be removed,

@jredrejo streamlining the experience so the admin goes straight to Device is helpful, but would we still need to keep it in, in case they decide to explore other places of the app before they start importing content? Unless content import is the only thing a user can do post-setup, might be good to keep `ContentUnavailablePage.vue` in.